lib/flow/Advection.cpp

```diff
@@ -678,7 +678,7 @@ float Advection::_applyPeriodic(float val, float min, float max) const
 
 void Advection::_printNonZero(int rtn, const char *file, const char *func, int line) const
 {
-#ifdef VPRINT
+#ifndef NDEBUG
```

Comment:
Prints hundreds of debug lines that are probably not necessary for every debug build.

Reply:
Well, I wasn't able to print out hundreds of lines debug output, but it's reverted in 3e4e4c7b0 .

Reply:
Might've been my session file. Thanks.